activities.css

```diff
@@ -0,0 +1,86 @@
+article
+{
+    margin-top: 60px;
+}
+
+#activity
+{
+    font-size: 18px;
+}
+
+#activities
+{
+    display: flex;
+    margin-left: -95px;
+}
+
+.bigSize
+{
+    border: 1px solid white;
+    border-radius: 10px;
+    box-shadow: 2px 2px 5px grey;
+    width: 357px;
+    margin-left: 110px;
+    margin-top: 60px;
+    height: 515px;
+}
+
+.bigImages
+{
+    border-radius: 10px 10px 0px 0px;
+    margin-left: -41px;
+    margin-top: -17px;
+    width: 360px;
+    height: 470px;
+}
+
+.colomn
+{
+    margin-top: -26px;
+}
+
+.textimages
+{
+    font-weight: bold;
+    margin-top: 10px;
+    margin-left: -15px;
+}
+
+.squaresB
+{
+    border: 1px solid white;
+    margin-top: 85px;
+    border-radius: 10px;
+    box-shadow: 2px 2px 5px grey;
+    margin-left: 110px;
+    width: 357px;
+
+}
+
+.squaresC
+{
+    border: 1px solid white;
+    margin-top: 85px;
+    border-radius: 10px;
+    box-shadow: 2px 2px 5px grey;
+    margin-left: 110px;
+    width: 357px;
+    height: 255px;
+}
+
+.mediumsize
+{
+    width: 360px;
+    margin-left: -42px;
+    margin-top: -17px;
+    border-radius: 10px 10px 0px 0px;
+}
+
+.mediumsizeChange
```

Comment:
as example here it's neither a cameCase nor  lowercase :p you have to decide a naming convention and stick to it :)
it will help you greatly in the futur like when you splitted the css in many different files.